From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Doe <john.doe@somewhere.on.planet>
Date: Tue, 4 Nov 2025 20:03:05 +0000
Subject: Patching kernel bcm2711 files drivers/usb/dwc2/gadget.c

Signed-off-by: John Doe <john.doe@somewhere.on.planet>
---
 drivers/usb/dwc2/gadget.c | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/drivers/usb/dwc2/gadget.c b/drivers/usb/dwc2/gadget.c
index 834fc0261..89557be9c 100644
--- a/drivers/usb/dwc2/gadget.c
+++ b/drivers/usb/dwc2/gadget.c
@@ -4342,12 +4342,16 @@ static int dwc2_hsotg_ep_dequeue(struct usb_ep *ep, struct usb_request *req)
 		spin_unlock_irqrestore(&hs->lock, flags);
 		return -EINVAL;
 	}
 
 	/* Dequeue already started request */
-	if (req == &hs_ep->req->req)
-		dwc2_hsotg_ep_stop_xfr(hs, hs_ep);
+	if (req == &hs_ep->req->req) {
+				dev_warn(hs->dev,
+					 "%s: DEQUEUE\n",
+					 __func__);
+		// dwc2_hsotg_ep_stop_xfr(hs, hs_ep);
+	}
 
 	dwc2_hsotg_complete_request(hs, hs_ep, hs_req, -ECONNRESET);
 	spin_unlock_irqrestore(&hs->lock, flags);
 
 	return 0;
-- 
Created with Armbian build tools https://github.com/armbian/build

